django-cloudlaunch/cloudlaunch/migrations/0007_auto_20181030_1316.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.1.2 on 2018-10-30 17:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cloudlaunch', '0006_add_multiple_api_keys'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='userprofile',
+            name='user',
+        ),
+        migrations.AlterField(
+            model_name='applicationdeployment',
+            name='archived',
+            field=models.BooleanField(blank=True, default=False),
+        ),
+        migrations.AlterField(
+            model_name='publickey',
+            name='default',
+            field=models.BooleanField(blank=True, default=False, help_text='If set, use as the default public key'),
+        ),
+        migrations.DeleteModel(
+            name='UserProfile',
```

Comment:
This migration is deleting the UserProfile? Does this change need a migration?

Reply:
Sorry, I just realised that the UserProfile has been accidentally duplicated. In that case, the UserProfile signal connection should probably be done in djcloudbridge, instead of here?

Reply:
I moved the signal functions to djcloudbridge (https://github.com/CloudVE/djcloudbridge/pull/2/commits/637c16b2f1f1577e75dbb9f7bc24c7c47db9cee5), but it seems that the connection has to happen in CloudLaunch (or at least I couldn't get it to work with the connection in any part of djcloudbridge). As recommended by the official docs, I made the connections in apps.py